pkgs/applications/video/kooha/default.nix

```diff
@@ -0,0 +1,61 @@
+{ stdenv, lib, fetchFromGitHub, appstream-glib, desktop-file-utils, glib
+, gobject-introspection, gsettings-desktop-schemas, gst_all_1, gtk3, libhandy
+, librsvg, meson, ninja, pkg-config, python3, python3Packages, wrapGAppsHook }:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "kooha";
+  version = "1.1.1";
+  format = "other";
+
+  src = fetchFromGitHub {
+    owner = "SeaDve";
+    repo = "Kooha";
+    rev = "v${version}";
+    sha256 = "05515xccs6y3wy28a6lkyn2jgi0fli53548l8qs73li8mdbxzd4c";
+  };
+
+  buildInputs = [
+    glib
+    gobject-introspection
+    gst_all_1.gstreamer
+    gst_all_1.gst-plugins-base
+    gtk3
+    libhandy
+    librsvg
+  ];
+
+  nativeBuildInputs = [
+    appstream-glib
+    desktop-file-utils
+    gst_all_1.gstreamer
+    gst_all_1.gst-plugins-base
+    gobject-introspection
```

Comment:
Why are they required twice?

Reply:
Apparently they aren't. So just leftover of me throwing the kitchen sink at trying to get it to work. Removed.